judgels-backends/uriel/uriel-app/src/test/java/judgels/uriel/contest/clarification/ContestClarificationResourceTests.java

```diff
@@ -0,0 +1,105 @@
+package judgels.uriel.contest.clarification;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import java.time.Instant;
+import java.util.Optional;
+import judgels.persistence.api.Page;
+import judgels.sandalphon.problem.ProblemClient;
+import judgels.service.actor.ActorChecker;
+import judgels.service.api.actor.AuthHeader;
+import judgels.uriel.api.contest.Contest;
+import judgels.uriel.api.contest.clarification.ContestClarification;
+import judgels.uriel.contest.ContestStore;
+import judgels.uriel.contest.log.ContestLogger;
+import judgels.uriel.contest.problem.ContestProblemStore;
+import judgles.jophiel.user.UserClient;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+class ContestClarificationResourceTests {
```

Comment:
It is intentional that in Judgels, we don't have unit tests for resource layer because it is very fragile and very much testing implementation details (we have to know all dependencies and all interaction between dependencies). Also the mocks make us not really confident that this actually testing the correct result that we want.

We have integration tests for resource layer in `ContestClarificationServiceIntegrationTests`, can we add some tests there instead? Here, we actually test the HTTP layer -- we test that the query param can actually be passed and test that Dropwizard can pass it through our registered Jersey resource, etc, which is far more useful test.

Reply:
Same as other comment, I can add the integration test anyway, and remove the unit tests if that's preferable.

Reply:
Updated, removed the unit test.